mem/basic.md

```diff
@@ -235,7 +236,7 @@ A size is valid, whenever it is non-negative and in-bounds for signed `PTR_SIZE`
 ```rust
 impl Memory for BasicMemory {
     fn valid_size(size: Size) -> bool {
-        size.in_bounds(Signed, Self::PTR_SIZE) && size >= 0
```

Comment:
I think it makes more sense to check that a Size is non-negative when constructing `Size`.

Reply:
I like that -- Size as 'unsigned BigInt', basically. Or as `BigNat`.

Can you update the `Size` docs accordingly?

Reply:
@RalfJung done!

Reply:
Please also add doc comments to `from_bits` and `from_bytes` saying that they will panic then the argument is negative.

Reply:
uh, as we are rounding up an input from `[-7, -1]` wouldn't even panic from_bits, I'll change that.

Reply:
Oh though got rounded up to 0? Ouch, good catch.